README.md

```diff
@@ -56,6 +56,14 @@ Then it can be used with `aio app test` or `npm test` by adding a test script to
     }
 ```
 
+### Debugging workers
+
+If you use the [debug](https://www.npmjs.com/package/debug) library inside your worker code, you can enable it in `test-worker` and `run-worker` commands by setting the `WORKER_DEBUG` environment variable just like the standard `DEBUG` variable:
+
+```
+WORKER_DEBUG=mystuff aio asset-compute test-worker
```

Comment:
maybe also say that `WORKER_DEBUG=mystuff*` is supported?

Reply:
I am linking to the `debug` library and mention the var works like the standard `DEBUG` var it uses. It probably doesn't make sense to repeat everything it supports since it will be difficult to keep updated.